apps/src/gamelab/GameLabP5.js

```diff
@@ -435,6 +646,7 @@ GameLabP5.prototype.getCustomMarshalBlockedProperties = function () {
 
 GameLabP5.prototype.getCustomMarshalObjectList = function () {
   return [
+    { instance: GameLabGame },
```

Comment:
I like what you've done here, it's a nice clean way of expressing this new object.  Can we retroactively apply this approach to anything else? (Maybe the camera?)